### ISSUE TITLE
dotenv can't handle empty lines or lines with comments

### ISSUE THREAD

**Issue by theronic** [LynxEyes/dotenv.clj#2]
If there is a blank space in your .env file, as is common when you have lots of environment variables and you group them, dotenv fails to load at compile-time.

**Comment by ivopt** [comment]
🤔 I see. Thank you for reporting.

Could you please be more explicit with the "dotenv fails to load at compile-time" part so I can have a look into that?

Thanks

**Comment by theronic** [comment]
Make a .env like this and try to require `[dotenv :refer [env app-env]]`:

```
A=123

B=abc
```

Execution error (IllegalArgumentException) at dotenv/load-env-file (dotenv.clj:17).
Vector arg to map conj must be a pair

```
(defn- load-env-file [filename]
  "loads an env file into a map"
  (if (.exists (io/as-file filename))
    (->> filename
         slurp
         to-pairs ;; caused by this
         (into {}) ) ;; issue here
    {}))
```

`(into {})` fails for empty lines because `to-pairs` splits on `=`.

**Comment by ivopt** [comment]
Cool, thanks for the example.
I'll push a fix as soon as I can (still today).

**Comment by theronic** [comment]
Thanks! Support for comments starting with # would also be convenient.

**Comment by ivopt** [comment]
I will take that into account, thanks for referring to the situation 😄

**Comment by ivopt** [comment]
Just fixed and pushed a new version 1.0.3 to clojars

https://clojars.org/lynxeyes/dotenv/versions/1.0.3

Thank you so much for pointing this out.
Any further situations, open new issues. Thanks!!
